onebusaway-android/src/main/java/com/joulespersecond/oba/ObaAnalytics.java

```diff
@@ -0,0 +1,260 @@
+/*
+ * Copyright (C) 2014
```

Comment:
For all new files, please change this to:
`... Copyright (C) 2014-2015 University of South Florida`

For existing files that were modified, if USF isn't already added in the copyright header, we should add it, and update the copyright date.  So, if it looked like this before:
`... Copyright (C) 2011 Paul Watts`

...we should change it to:
`... Copyright (C) 2011-2015 Paul Watts, University of South Florida`